### PR TITLE
update APIServerTracing docs

### PR DESCRIPTION
**Pull request by logicalhan**
Enhancement issue: https://github.com/kubernetes/enhancements/issues/647

### CODE REVIEW

File: content/en/docs/concepts/cluster-administration/system-traces.md [hunk 1]
```diff
@@ -59,22 +59,20 @@ as the kube-apiserver is often a public endpoint.
...
+apiVersion: apiserver.config.k8s.io/v1beta1
 kind: TracingConfiguration
 # default value
 #endpoint: localhost:4317
 samplingRatePerMillion: 100
 ```
 
 For more information about the `TracingConfiguration` struct, see
-[API server config API (v1alpha1)](/docs/reference/config-api/apiserver-config.v1alpha1/#apiserver-k8s-io-v1alpha1-TracingConfiguration).
+[API server config API (v1beta1)](/docs/reference/config-api/apiserver-config.v1beta1/#apiserver-k8s-io-v1beta1-TracingConfiguration).
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/en/docs/concepts/cluster-administration/system-traces.md:75]: PR #40330 proposed.

[the remark this answers is not in the document]
Comment (tengqm) [reply content/en/docs/concepts/cluster-administration/system-traces.md:75]: Double checked the v1beta1 API and found that `TracingConfiguration` is missing from it.
The `TracingConfiguration` type was added on 2023-03-07, so it is only part of the API in 1.27.
This means the link will still be dangling. We will make sure it works when 1.27 is out of the door.

File: content/en/docs/concepts/cluster-administration/system-traces.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ weight: 90
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.27" state="beta" >}}
```
Comment (tengqm) [inline content/en/docs/concepts/cluster-administration/system-traces.md:12]: You will need to update the `content/en/docs/reference/command-line-tools-reference/feature-gates.md` file to indicate that `APIServerTracing` is now beta.

File: content/en/docs/concepts/cluster-administration/system-traces.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (logicalhan) [reply content/en/docs/concepts/cluster-administration/system-traces.md:75]: lgtm

**Review by dgrisonnet — APPROVED** [review]
/lgtm

**Pull request merged**